packages/logrotate/build

```diff
@@ -21,9 +21,9 @@ nomail
 
 /var/lib/dcos/mesos/log/* {
     olddir /var/lib/dcos/mesos/log/archive
-    maxsize 2000k
-    daily
-    rotate 7
+    maxsize 256M
```

Comment:
We must cap size. We run on many hosts with very small root disks (AWS instances), and have customers run out of disk. It's why we added the logrotate in the first place.